mail/1.6/mail-spec-1.6.html

```diff
@@ -0,0 +1,664 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="UTF-8">
+<!--[if IE]><meta http-equiv="X-UA-Compatible" content="IE=edge"><![endif]-->
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<meta name="generator" content="Asciidoctor 1.5.8">
+<meta name="author" content="Jakarta Mail Team, https://projects.eclipse.org/projects/ee4j.mail">
```

Comment:
Project link seems to actually be at: https://projects.eclipse.org/projects/ee4j.javamail